configuration/restdocs.md

```diff
@@ -7,14 +7,38 @@ title: openHAB REST API
 
 # openHAB REST API
 
-The REST API of openHAB serves different purposes.
-It can be used to integrate openHAB with other systems as it allows read access to items and item states as well as status updates or the sending of commands for items.
-Furthermore, it gives access to the sitemaps, so that it is the interface to be used by remote user interfaces (e.g. fat clients or javascript based web clients).
-The REST API also supports server-push, so you can subscribe yourself on change notification for certain resources.
+Through the openHAB [REST API](https://en.wikipedia.org/wiki/REST_API) most aspects of the openHAB system can be accessed by other programs.  
+This includes for example the access to all data related to Items, Things and Bindings as well as the capabilities to invoke  actions that change the state of Items. Interactions with the REST API are based on a simple http protocoll.
+The REST API provides an interface that can only be accessed programmatically.
+The information available from the API includes Items and their state, installed bindings, sitemap information, and much more. 
+The information exchange takes place through HTTP and the API provides a defined protocoll for information exchange among different computer systems. 
+However, from a technical perspective it is very easy to use. 
+Be aware that the REST API is not automatically installed in openHAB. 
+Access over the Internet to the REST API is possible but this represents a significant security risk, and users are encouraged to [ensure safe and secure connections](http://docs.openhab.org/installation/security.html). 
+
+## Examples for the Use of the REST API
+Some tasks that can be accomplished by using the REST API include:
+
+- Using third party software on cell phones, such as tasker to [open your garage door](https://community.openhab.org/t/triggering-items-using-openhab-2s-rest-api-from-tasker/14027)
+- Interacting with openHAB via other programs; several computer languages offer libraries to easily access REST APIs generally, including openHAB’s REST API
```

Comment:
Yuck - I wrote it, I don't like it.  Consider revising "several computer languages offer libraries to easily access REST APIs generally, including openHAB's REST API" to:

many computer languages can easily make use of REST APIs